packages/@sanity/desk-tool/src/pane/styles/EmptyListPane.css

```diff
@@ -0,0 +1,8 @@
+@import 'part:@sanity/base/theme/variables-style';
+
+.root {
+}
+
+.message {
```

Comment:
This CSS needs to work for all screen widths

Reply:
What does this comment mean? Is it still work left that needs to be done?

Reply:
Yes. In my opinion, at least